### PR TITLE
Add Rate Limits section under Storage

### PR DESCRIPTION
**Pull request by aayushmahapatra**

### CODE REVIEW

File: pages/storage/rate-limits.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+- **Pro:** Our Pro plan is designed for professionals and businesses that demand higher bandwidth and storage capacities. With ample resources for IPFS and Arweave, this plan accommodates larger projects and organizations.
+- **Wagmi:** For enterprises and high-demand scenarios, our Wagmi plan offers substantial bandwidth and storage, along with an extensive range of services. This plan is tailored for those who require top-tier performance and support.
+
+Each plan comes with its own set of rate limits, ensuring fair usage and optimal performance. Explore the details of each plan to choose the one that aligns with your specific needs.
+
+|                                  | **Default** | **Lite** | **Pro** | **Wagmi** |
+| -------------------------------- | ----------- | -------- | ------- | --------- |
+| **PRICE**                        | 0           | 3        | 10      | 50        |
+| **CDN Bandwidth (GB)**           | 100         | 100      | 100     | 100       |
```
[the remark this answers is not in the document]
Comment (rekpero) [reply pages/storage/rate-limits.mdx:33]: same with others

File: pages/storage/rate-limits.mdx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+| **IPFS Bandwidth (GB)**          | 10          | 100      | 400     | 2500      |
+| **Storage Arweave**              | 0           | 300 MB   | 1 GB    | 3 GB      |
+| **Storage IPFS (GB)**            | 5           | 40       | 150     | 800       |
+| **Storage Filecoin (GB)**        | 5           | 40       | 150     | 800       |
+| **Domains**                      | 9999        | 9999     | 9999    | 9999      |
+| **Hns Domains**                  | 2           | 3        | 5       | 10        |
+| **Ens Domains**                  | 2           | 3        | 5       | 10        |
+| **ipfsNumberOfRequests**         | 1E+18       | 1E+18    | 1E+18   | 1E+18     |
+| **Number of Requests**           | 1E+18       | 1E+18    | 1E+18   | 1E+18     |
+| **Optimized Images**             | 25          | 100      | 500     | 2000      |
```
Comment (rekpero) [inline pages/storage/rate-limits.mdx:43]: remove these

File: pages/storage/rate-limits.mdx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+| **Parallel Uploads**             | 5           | 25       | 50      | 100       |
+| **IPFS Gateway Service Domains** | 9999        | 9999     | 9999    | 9999      |
```
Comment (rekpero) [inline pages/storage/rate-limits.mdx:45]: change this title to `IPFS Gateway`- put infinity icon

File: pages/storage/rate-limits.mdx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+- **Default:** Our entry-level plan, designed for those getting started. It offers a generous amount of CDN and IPFS bandwidth, making it ideal for small projects and experimentation. Plus, it's entirely free!
+- **Lite:** The Lite plan is perfect for individuals or small businesses with moderate requirements. It provides increased CDN and IPFS bandwidth, additional storage on Arweave, and extended support for various domains.
+- **Pro:** Our Pro plan is designed for professionals and businesses that demand higher bandwidth and storage capacities. With ample resources for IPFS and Arweave, this plan accommodates larger projects and organizations.
+- **Wagmi:** For enterprises and high-demand scenarios, our Wagmi plan offers substantial bandwidth and storage, along with an extensive range of services. This plan is tailored for those who require top-tier performance and support.
+
+Each plan comes with its own set of rate limits, ensuring fair usage and optimal performance. Explore the details of each plan to choose the one that aligns with your specific needs.
+
+|                                  | **Default** | **Lite** | **Pro** | **Wagmi** |
```
Comment (rekpero) [inline pages/storage/rate-limits.mdx:30]: also wait where is the rate limit in here like how much file size can be uploaded

**Review by rekpero — CHANGES_REQUESTED** [review]
check my comments

File: pages/storage/rate-limits.mdx (hunk 4 above)
Comment (aayushmahapatra) [reply pages/storage/rate-limits.mdx:30]: It's mentioned like - `Storage IPFS`, `Storage Filecoin`

File: pages/storage/rate-limits.mdx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+- **Pro:** Our Pro plan is designed for professionals and businesses that demand higher bandwidth and storage capacities. With ample resources for IPFS and Arweave, this plan accommodates larger projects and organizations.
+- **Wagmi:** For enterprises and high-demand scenarios, our Wagmi plan offers substantial bandwidth and storage, along with an extensive range of services. This plan is tailored for those who require top-tier performance and support.
+
+Each plan comes with its own set of rate limits, ensuring fair usage and optimal performance. Explore the details of each plan to choose the one that aligns with your specific needs.
+
+| **Resource**              | **Storage Default** | **Storage Lite** | **Storage Pro** | **Storage Wagmi** |
+| ------------------------- | ------------------- | ---------------- | --------------- | ----------------- |
+| **PRICE**                 | $0                  | $3               | $10             | $50               |
+| **CDN Bandwidth (GB)**    | 100 GB              | 100 GB           | 100 GB          | 100 GB            |
```
Comment (rekpero) [inline pages/storage/rate-limits.mdx:36]: Also add emoji 💲 on - 
CDN Bandwidth, Storage Arweave, HNS, ENS

Also rename Storage Arweave to Arweave Storage, same with IPFS

File: pages/storage/rate-limits.mdx [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+| **IPFS Bandwidth (GB)**   | 10 GB               | 100 GB           | 400 GB          | 2500 GB           |
+| **Storage Arweave**       | 0 MB                | 300 MB           | 1 GB            | 3 GB              |
+| **Storage IPFS (GB)**     | 5 GB                | 40 GB            | 150 GB          | 800 GB            |
+| **Storage Filecoin (GB)** | 5 GB                | 40 GB            | 150 GB          | 800 GB            |
```
Comment (rekpero) [inline pages/storage/rate-limits.mdx:40]: remove this

Comment (rekpero) [reply pages/storage/rate-limits.mdx:40]: Add this note at the end say - `By default, Filecoin storage utilizes IPFS storage. Therefore, storage consumption is capped at the IPFS storage limit. In simpler terms, you can upload until you reach the IPFS storage capacity.`

**Review by rekpero — CHANGES_REQUESTED** [review]
check my comments

File: pages/storage/rate-limits.mdx (hunk 4 above)
Comment (rekpero) [reply pages/storage/rate-limits.mdx]: I'm talking about `File upload size / Request` rate limit. What you have mostly mention is about the plan and all. But mention this as well. check with milos for this

**Review by rekpero — APPROVED** [review]
